packages/button/src/Button.tsx

```diff
@@ -1,16 +1,12 @@
 import React, {
 	forwardRef,
-	DetailedHTMLProps,
 	ButtonHTMLAttributes,
 	AnchorHTMLAttributes,
 } from 'react';
 import { useLinkComponent } from '@ag.ds-next/core';
 import { buttonStyles, ButtonSize, ButtonVariant } from './styles';
 
-export type ButtonProps = DetailedHTMLProps<
-	ButtonHTMLAttributes<HTMLButtonElement>,
-	HTMLButtonElement
-> & {
+export type ButtonProps = ButtonHTMLAttributes<HTMLButtonElement> & {
```

Comment:
Updating this type here as per Mitchells suggestion. Will update other components in another PR.